### PR TITLE
Added meteor packaging option

### PR DESCRIPTION
**Pull request by dotansimha**
Hi ExactTarget,

I'm a package maintainer for Meteor.js, a popular full-stack JavaScript framework.
I'm also a developer on angular-meteor, a library that let's you work with AngularJS on top of Meteor.

A lot of our users use fuelux and it right now there is a separate package and Github repo that simply manually copies your distribution and publish it to Meteor.
This PR will allow you to directly publish updated versions of the library as they become available. All you have to do is create an account at https://meteor.com/ (click SIGN IN, then Create account). After you've done that, please let me know the name of the account, and I'll add you as a maintainer to the "ExactTarget" organization there.

I've already published the current version of the package on Atmosphere (Meteor's package directory). When you release new versions of the library, you'll be able to publish them to Atmosphere by running "grunt meteor-publish".

Thanks & best regards,
Dotan.

### CODE REVIEW

**Comment by interactivellama** [comment]
Thanks for the PR and instructions. Will look into to it.

**Comment by swilliamset** [comment]
@dotansimha this looks great. we would love to help make the meteor/fuelux experience smoother. we'll review your changes and look at integrating a `meteor-publish` step to our release process. 

But first could we ask for some modifications to your PR? There appears to be a number of whitespace and code style modifications which were most likely unintended. Please take a look at your changes with the white list query param and attempt to resolve these.

https://github.com/ExactTarget/fuelux/pull/1258/files?w=1

**Comment by vernak2539** [comment]
:+1:

**Comment by swilliamset** [comment]
Closing until @dotansimha provides an update

File: Gruntfile.js [hunk 1]
```diff
@@ -72,8 +69,19 @@ module.exports = function (grunt) {
...
-			zipsrc: ['dist/fuelux'],// temp folder
-			screenshots: ['page-at-timeout-*.jpg']
+			zipsrc: ['dist/fuelux'], // temp folder
+			screenshots: ['page-at-timeout-*.jpg'],
+			meteor: ['.build.*', 'versions.json']
+		},
+		exec: {
+			'meteor-init': {
+				command: [
+          'type meteor >/dev/null 2>&1 || { curl https://install.meteor.com/ | sh; }'
```
Comment (swilliamset) [inline Gruntfile.js]: Don't feel comfortable running a curl in our grunt tasks

**Pull request closed without merging**